lib/features/popup-menu/PopupMenuComponent.js

```diff
@@ -77,6 +77,10 @@ export default function PopupMenuComponent(props) {
         return (entry.rank || 0) >= 0;
       }
 
+      if (entry.searchable === false) {
+        return 0;
```

Comment:
```suggestion
        return false;
```

Let's keep this interface uniform. We always return a `boolean`.

Reply:
You're right, got momentarily confused with the `0` from above. Updated